### DISCUSSION TITLE
<title>rate limit exceeded for plan FREE_EDUCATIONAL (Github Copilot

### DISCUSSION THREAD

**Discussion by joepalexander** [community/community#43840] [Copilot Conversations]
### Select Topic Area

Bug

### Body

getting the error
`[ERROR] [fetch] [2023-01-10T12:37:48.122Z] Unhandled status from server:,429,{"error":{"message":"rate limit exceeded","internal_message":"rate limit exceeded for plan FREE_EDUCATIONAL","type":"client_error"}}
` when using github copilot on the student plan

**Comment by SyahrulApr86** [comment]
i have the same issue

**Comment by retrievaaaa** [comment]
Yeah its pretty annoying, I have to rewrite the line like 4 times before I get any suggestions. Pretty much makes it unusable.

**Comment by lrdcxdes** [comment]
same

**Comment by giswqs** [comment]
Same here since yesterday

**Comment by Cwarcup** [comment]
Issue mentioned in #[43841](https://github.com/orgs/community/discussions/43841#discussioncomment-4649573).

Hopefully this can get resolved soon!

**Comment by ReaVNaiL** [comment]
Answered here in #43673 [Support Team](https://github.com/orgs/community/discussions/43673#discussioncomment-4655830)

**Comment by Akash1134** [comment]
Hi there,

Some details and update on this issue:

This week, we've had to take down some of our Copilot resources for maintenance. While we did our best to source extra capacity elsewhere, we've been forced to limit some free users for the period of time where one of our busiest models is out of commission.

We expect that this will cease to impact your use of Copilot after the 16th of January. In the meantime, you will have less frequent suggestions, but you should still be getting suggestions - just with rate limits in between. You may see these errors specifically if you look in the output pane or logs for your IDE.

Thanks for your patience and apologies for the inconvenience caused.
